docs/hardware/babble.mdx

```diff
@@ -0,0 +1,56 @@
+# Project Babble (Face Tracking)
```

Comment:
Consider removing `(Face Tracking)`

Reply:
Resolved in #[c2063f9](https://github.com/VRCFaceTracking/docs/pull/15/commits/c2063f9a32e0e1e23ea8fa10e6a546cc599908c9).